src/Tablebot/Plugins/Shibe.hs

```diff
@@ -0,0 +1,78 @@
+-- |
+-- Module      : Tablebot.Plugins.Shibe
+-- Description : A very simple plugin that provides shibe pictures.
+-- License     : MIT
+-- Maintainer  : tagarople@gmail.com
+-- Stability   : experimental
+-- Portability : POSIX
+--
+-- This is an example plugin which just responds with a shibe photo to a .shibe call
+module Tablebot.Plugins.Shibe (shibePlugin) where
+
+import Control.Monad.IO.Class (MonadIO (liftIO))
+import Data.Aeson (FromJSON, eitherDecode)
+import Data.Functor ((<&>))
+import Data.Text (Text, pack)
+import GHC.Generics (Generic)
+import Network.HTTP.Conduit (Response (responseBody), parseRequest)
+import Network.HTTP.Simple (httpLBS)
+import Tablebot.Plugin.Discord (Message, sendMessage)
+import Tablebot.Plugin.SmartCommand (parseComm)
+import Tablebot.Plugin.Types
+  ( Command,
+    DatabaseDiscord,
+    EnvCommand (Command),
+    EnvPlugin (..),
+    HelpPage (HelpPage),
+    Plugin,
+    RequiredPermission (None),
+    plug,
+  )
+
+-- | @ShibeAPI@ is the basic data type for the JSON object that the Shibe API returns
+data ShibeAPI = ShibeAPI
+  { url :: !Text
+  }
+  deriving (Show, Generic)
+
+instance FromJSON ShibeAPI
```

Comment:
```suggestion
type ShibeAPI = Text
```
Shibe only returns a raw URL, while the `url :: !Text` pattern is for a key-value pair `url: ...`.